collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: book
+title:  "The Enterprise Path to Service Mesh Architectures 2nd Edition"
+date:   2018-06-11 21:15:05 +0000
+image: /assets/images/books/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition.png
+cover: /assets/images/books/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition-small.png
+#author: Lee Calcote
+status: published
+brief: "Planning to build a microservice-driven cloud native application or looking to modernize existing application services?"
+description: "A must-read for anyone looking to understand service mesh deployments and get an explanation of service meshes in contrast to related technologies.
+Learn from the largest collection of service mesh projects and their maintainers in the world.Self-study service mesh workshops and tutorials for Istio, Envoy, Linkerd, Consul, App Mesh, Octarine, Kuma, VMware Tanzu Service Mesh"
+redirect_from: 
```

Comment:
We can safely remove this redirect. This duplicate will cause conflict Withrow eh existing redirect.